jekyll/_style/style-images.adoc

```diff
@@ -0,0 +1,12 @@
+= Using images
```

Comment:
Wondering if this should be a larger Images page, with a subsection dedicated to screenshots instead of having it on a separate page.

Reply:
Yeah that sounds good to me